### ISSUE TITLE
GetSurfaceHeight bug

### ISSUE THREAD

**Issue by Sigma88** [Kopernicus/Kopernicus#246]
**DISCLAIMER: I know this is not a Kopernicus bug, but since [Kopernicus uses that method](https://github.com/Kopernicus/Kopernicus/blob/master/src/Kopernicus/RuntimeUtility/RuntimeUtility.cs#L582) I think it's worth mentioning it here**

using PQS.GetSurfaceHeight(Vector3) does not return the correct value.

as far as I can tell, this is caused by anything with "ridged" noise

for example, if we look at the first PQSCity2 on Kerbin which has:
> (lat = -11.95, lon = 33.74, pos = [5396701, -1373531.5, 3604598])

we get the following results:

| method used | stock Kerbin | SigmaDimensions 10x | SigmaDimensions 20x<br>(no ridged crap) |
|---------|--------|---------|----------|
| <b>GetSurfaceHeight()</b> | 2651.62151 | 26516.19917874 | 21098.45243442 | 
| <b>landed kerbal</b> | 2638.22783068579 | 26378.4273154484 | 21094.3161698328 |
| <b>offset</b> | 13.39367958 | 137.7718633 | 4.136265 |
| <b>relative offset</b> | 13.39367958 | 13.77718633 | 0.20681325 |

as you can see the relative offset (the offset between GetSurfaceHeight() and "landed kerbal" all divided by the rescale) is basically zero when I remove the two PQSMods that have ridged crap on kerbin

```
@Kopernicus
{
	@Body[Kerbin]
	{
		@Template
		{
			removePQSMods = VertexRidgedAltitudeCurve,VertexHeightNoiseVertHeightCurve2
		}
	}
}
```

**Comment by Sigma88** [comment]
I tried adding `VertexHeightNoise` with `noiseType = RidgedMultifractal` to kerbin but it seems like that one doesn't mess up GetSurfaceHeight

**Comment by Sigma88** [comment]
I tried changing some numbers and I found out that the culprit is "frequency"

starting from lower values it works up untill 79.
at 80 the bug appears

**Comment by Sigma88** [comment]
solved with 6a1ce668bac3503cc87086a1513b3f6be76d142e
